imod/prepare/wells.py

```diff
@@ -25,21 +25,42 @@ def vectorized_overlap(bounds_a, bounds_b):
     )
 
 
+def compute_point_filter_overlap(bounds_wells, bounds_layers):
```

Comment:
Could you add type hints?

It makes it a bit hard to review not knowing what the input and output of this method is